### DISCUSSION TITLE
Running django oauth but it seems there are no errors but it goes to process_error in linkedin.py

### DISCUSSION THREAD

**Discussion by allison9y** [python-social-auth/social-core#1349] [Q&A]
I am using django oauth but it goes to 'process_error' with no serviceErrorCode when it seems things are okay.  I printed out the data which passed into process_error. 

    def process_error(self, data):
        super(LinkedinOAuth2, self).process_error(data)
        print(data)
        if data.get('serviceErrorCode'):
            raise ....

 <QueryDict: {'code': ['---code_value---'], 'state': ['hS---state---Cj']}>
 {'access_token': '---token---', 'expires_in': 5555555}

Btw, just paste the settings for oAuth2

SOCIAL_AUTH_LINKEDIN_OAUTH2_SCOPE = ['r_liteprofile', 'r_emailaddress']
SOCIAL_AUTH_LINKEDIN_OAUTH2_FIELD_SELECTORS = ['emailAddress']
SOCIAL_AUTH_LINKEDIN_OAUTH2_EXTRA_DATA = [
                                   ('id', 'id'),
                                   ('firstName', 'first_name'),
                                   ('lastName', 'last_name'),
                                   ('emailAddress', 'email'),
                                   ]

What's wrong here? Thanks.

**Comment by nijel** [comment]
`process_error` is called to detect error in the response, so there is nothing wrong it is being called.
